docs/webui_tutorial.rst

```diff
@@ -46,12 +46,16 @@ Below is a screenshot of the landing page.
 One last thing that might be interesting to note is that the address of the running Raiden node is always displayed in the top bar.
 
 
-Wallet
+Tokens
```

Comment:
Hijacking this to comment that it looks like channels page image has a weird box on it. Looks like some debug tool. In the top left corner. Please fix that.

Reply:
I took the screenshots with the debug window open to force a specific size but some how missed that. Thanks.